recipe/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "cytosnake" %}
```

Comment:
hmm, typically the bioconda recipes belong in the bioconda github org. I don't think adding it here makes sense because you will then need to maintain two versions? Plus, it is nonfunctional here. Maybe this process has changed?

Reply:
You're correct about not having it here. I just left it up here for all of you to see if it looks okay.

Reply:
Would it make sense to single-source this file here in this repo and utilize CI/CD tools to automatically synchronize the Bioconda version of the same file (for example, triggering on new repo releases)? This might deduplicate efforts and decrease the chance for things to become out of sync.

Reply:
I didn't know you can do that! added in #88